src/app/home/home.module.ts

```diff
@@ -14,6 +15,7 @@ import { WorkItemWidgetModule } from './work-item-widget/work-item-widget.module
 
 @NgModule({
   imports: [
+    AddSpaceOverlayModule,
```

Comment:
Don't need to import AddSpaceOverlayModule here.

Reply:
removed